services/QuillLMS/client/app/bundles/Evidence/components/activitySlides/explanationData.ts

```diff
@@ -4,7 +4,7 @@ export const explanationData = {
     header: "This activity is not graded",
     imageData: {
       imageAlt: "An illustration of an A+ that is crossed out",
-      imageUrl: "images/comprehension/no-grades.svg"
+      imageUrl: "images/evidence/no-grades.svg"
```

Comment:
These all seem like renames.  Is it worth removing the old SVG files from the `quill-cdn` directory so we don't have them taking up space?

Reply:
yeah good call, I'll remove them!